boards/arm/imxrt/teensy-4.x/Kconfig

```diff
@@ -0,0 +1,36 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+if ARCH_BOARD_TEENSY_4X
+choice
+	prompt "Boot Flash"
+	default TEENSY4_QSPI_FLASH
+
+config TEENSY4_QSPI_FLASH
+	bool "QSPI Flash"
+
+endchoice # Boot Flash
+
+choice
+	prompt "Board configuration"
+
+config TEENSY_40
+	bool "Teensy 4.0"
+
+config TEENSY_41
+	bool "Teensy 4.1"
+
+endchoice # Board configuration
+
+menu "Sensors interfaces"
+
+config BMP280_LPI2C
```

Comment:
I don't think this should be here.  I don't see it on the board schematics.

Reply:
@btashton Thanks for your review! Agreed, this was more like a relict from my testing and experimenting.